pkg/cloud/scope/machine.go

```diff
@@ -381,3 +379,107 @@ func (m *MachineScope) SetInterruptible() {
 		m.AWSMachine.Status.Interruptible = true
 	}
 }
+
+// Copied from https://github.com/kubernetes-sigs/cluster-api/blob/bda002f52575eeaff68da1ba33c8ef27d5b1014c/controllers/noderefutil/providerid.go
```

Comment:
We don't need to copy this from upstream.

Reply:
Same as above

Reply:
FYI https://github.com/kubernetes-sigs/cluster-api/tree/v1.6.0-beta.1/controllers/noderefutil